tasks/main.yml

```diff
@@ -77,5 +77,20 @@
     name: "{{ item }}"
     state: started
     enabled: yes
+    use: "service"
   with_items: "{{ varnish_enabled_services | default([]) }}"
-  when: varnish_enabled_services
+  when: >
+    varnish_enabled_services and
+    (ansible_os_family != 'Debian' and ansible_distribution_release != "xenial")
+
+# See: https://github.com/ansible/ansible/issues/22303
+- name: Ensure Varnish services are started enabled on startup (Xenial specific)
+  service:
+    name: "{{ item }}"
+    state: started
+    enabled: yes
+    use: "service"
```

Comment:
ehhrm good catch. The initial one shouldn't use `service` -- instead, it should just use the Ansible default init system, in case the shim for `sysvinit` disappears at some point. I'll fix that up.